docs/core/whats-new/dotnet-10/overview.md

```diff
@@ -0,0 +1,100 @@
+---
+title: What's new in .NET 10
+description: Learn about the new features introduced in .NET 10 for the runtime, libraries, and SDK. Also find links to what's new in other areas, such as ASP.NET Core.
+titleSuffix: ""
+ms.date: 02/20/2025
+ms.topic: whats-new
+ai-usage: ai-assisted
+---
+
+# What's new in .NET 10
+
+Learn about the new features in .NET 10 and find links to further documentation. This page has been updated for Preview 1.
+
+.NET 10, the successor to [.NET 9](../dotnet-9/overview.md), has a special focus on cloud-native apps and performance. It will be [supported for 3 years](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a long-term support (LTS) release. You can [download .NET 10 here](https://get.dot.net/10).
```

Comment:
```suggestion
.NET 10, the successor to [.NET 9](../dotnet-9/overview.md) and will be [supported for 3 years](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a long-term support (LTS) release. You can [download .NET 10 here](https://get.dot.net/10).
```

I think we should remove the focus areas. We should leave that style of positioning to blog posts (and haven't done so yet).